src/app/app.component.ts

```diff
@@ -110,11 +110,11 @@ export class AppComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   openReportInSeparateTab(data: ReportData): void {
-    const tabIndex: number = this.tabs.findIndex((tab: Tab): boolean => tab.id === data.report.storageId);
+    const tabIndex: number = this.tabs.findIndex((tab: Tab): boolean => tab.id === String(data.report.storageId));
```

Comment:
Maybe it's best if `id` in `Tab` is also of type number

Reply:
I'm afraid that's gonna have to get it's own PR for a different refactor since this PR will probably get too big otherwise. The createCompareTabId method from the helperService returns an odd tabId that can't be converted to a number
```
createCompareTabId(originalReport: Report, runResultReport: Report): string {
    return `${originalReport.storageId}-${runResultReport.storageId}`;
 }
  ```
  I can't take a quick solution for this for now so maybe it's best to add it as an issue to the refactor board. What do you think?

Reply:
We can leave it for now.